src/stream/src/common/table/state_table.rs

```diff
@@ -351,7 +360,7 @@ where
             )
         };
 
-        let op_consistency_level = if is_consistent_op {
+        let op_consistency_level = if is_consistent_op && !crate::consistency::insane() {
```

Comment:
Please explain the relationship between `op_consistency_level` and `insane` in comment